Codigo/General.bas

```diff
@@ -44,81 +44,46 @@ Public Declare Sub OutputDebugString Lib "kernel32" Alias "OutputDebugStringA" (
 
 Global LeerNPCs As New clsIniManager
 
-Sub DarCuerpoDesnudo(ByVal UserIndex As Integer)
-        
-        On Error GoTo DarCuerpoDesnudo_Err
-        
-
-        '***************************************************
-        'Autor: Nacho (Integer)
-        'Last Modification: 03/14/07
-        'Da cuerpo desnudo a un usuario
-        '***************************************************
-        Dim CuerpoDesnudo As Integer
-
-100     Select Case UserList(UserIndex).genero
 
+Sub SetNakedBody(ByRef user As t_User)
+        user.flags.Desnudo = 1
+        Select Case user.genero
             Case e_Genero.Hombre
-
-102             Select Case UserList(UserIndex).raza
-
-                    Case e_Raza.Humano
-104                     CuerpoDesnudo = 21 'ok
-
-106                 Case e_Raza.Drow
-108                     CuerpoDesnudo = 32 ' ok
-
-110                 Case e_Raza.Elfo
-112                     CuerpoDesnudo = 510 'Revisar
-
-114                 Case e_Raza.Gnomo
-116                     CuerpoDesnudo = 508 'Revisar
-
-118                 Case e_Raza.Enano
-120                     CuerpoDesnudo = 53 'ok
-
-122                 Case e_Raza.Orco
-124                     CuerpoDesnudo = 248 ' ok
-
+               Select Case user.raza
```

Comment:
it could be nice to have this values as const or even better to have a predefined array with the values for each element if e_Raza preset

Reply:
to be done in a new patch, I agree magic numbers are bad